Python_package/prg/prg.py

```diff
@@ -10,6 +10,11 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
+
+def alen(x):
+    return 1 if np.isscalar(x) else len(x)
+
```

Comment:
Might be better to use shape[0] to be clearer that this is supposed to return the length of the first dim.